examples/python/lift.linux-x86_64.json

```diff
@@ -8,33 +8,30 @@
           "": {
             "description": "The Pants build tool.",
             "env": {
-              "=PATH": "{cpython}/python/bin:{scie.env.PATH}",
               "PEX_.*": null
             },
-            "exe": "{scie.bindings.venv}/venv/bin/python3.9",
-            "args": [
-              "{scie.bindings.venv}/venv/pex"
-            ]
+            "exe": "{scie.bindings.venv}/venv-3.{scie.env.PYTHON_MINOR=9}/pex"
           },
           "repl": {
             "description": "A Python repl with Pants (minus plugins) available for inspection.",
             "env": {
-              "=PATH": "{cpython}/python/bin:{scie.env.PATH}",
               "=PYTHONPATH": null
             },
-            "exe": "{scie.bindings.venv}/venv/bin/python3.9"
+            "exe": "{scie.bindings.venv}/venv-3.{scie.env.PYTHON_MINOR=9}/bin/python"
           }
         },
         "bindings": {
           "venv": {
             "description": "Installs Pants in a venv and pre-compiles .pyc.",
             "env": {
-              "=PATH": "{cpython}/python/bin:{scie.env.PATH}",
               "PEX_TOOLS": "1",
               "PEX_ROOT": "{scie.bindings}/pex_root",
-              "PEX_.*": null
+              "PEX_.*": null,
+              "=__PYTHON_3_8": "{cpython38}/python/bin/python3.8",
+              "=__PYTHON_3_9": "{cpython39}/python/bin/python3.9",
+              "=__PYTHON": "{scie.env.__PYTHON_3_{scie.env.PYTHON_MINOR=9}}"
             },
-            "exe": "{cpython}/python/bin/python3.9",
+            "exe": "{scie.env.__PYTHON}",
```

Comment:
Pretty neat!

Reply:
All right, this took a good bit of fiddling to test out over in a-scie/lift to verify it meets all of scie-pants fairly advanced usage / needs of the scie-jump lift format. This enhancement does turn out to suffice.